### DISCUSSION TITLE
symlink doubt

### DISCUSSION THREAD

**Discussion by araemis** [sxyazi/yazi#1266] [1. Q&A]
it supposed to be working or is just me that it dont
{ on = [ "-" ],         run = "link",                       desc = "Symlink the absolute path of files" },
{ on = [ "_" ],         run = "link --relative",            desc = "Symlink the relative path of files" },

sorry if its no the right place for the question

**Comment by AnirudhG07** [comment]
Yes it does work. It works on yanked files. So to use this
1) Yank a file/files
2) Go to some other directory
3) press "-" for absolute path based symlink or "_" for relative

If you try to symlink two files with the keybindings, it wont work.

Lemme know if this doesn't work out for you.

**Reply by araemis** [reply]
thanks, i think i supposed to press "-" at file that i wanna make a sym and would pop some msg until i choose the dir ti paste the sym
